### PR TITLE
PYIC-2876: add face to face multiple doc check page

### PR DESCRIPTION
**Pull request by shivanshuit914**
### What changed

- Added face to face multiple doc check page.

### Why did it change

Screenshot 2023-07-18 at 14 30 32

### Issue tracking

- [PYIC-2876](https://govukverify.atlassian.net/browse/PYIC-2876)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-2876]: https://govukverify.atlassian.net/browse/PYIC-2876?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/views/ipv/page-f2f-multiple-doc-check.njk [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  <p class="govuk-body">{{'pages.pageMultipleF2FDocCheck.content.paragraph5' | translate | safe }}</p>
+  <p class="govuk-body">{{'pages.pageMultipleF2FDocCheck.content.paragraph6' | translate | safe }}</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>{{'pages.pageMultipleF2FDocCheck.content.bullet3' | translate | safe }}</li>
+    <li>{{'pages.pageMultipleF2FDocCheck.content.bullet4' | translate | safe }}</li>
+  </ul>
+
+  <h2 class="govuk-heading-m">{{'pages.pageMultipleF2FDocCheck.content.subHeading2' | translate }}</h2>
+
+  <form id="passportMultipleDocCheckingForm" action="/ipv/page/{{pageId}}" method="POST" onsubmit="return passportMultipleDocCheckingFormSubmit()">
```
Comment (AmritSidhu) [inline src/views/ipv/page-f2f-multiple-doc-check.njk]: Should change the name of the form here

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -275,6 +275,30 @@
...
+        "bullet1": "have a UK passport or UK driving licence",
+        "bullet2": "can answer some security questions about things like your mobile phone contract, and any bank accounts, credit cards, loans or mortgages you may have",
+        "paragraph2": "It takes about 10 minutes to prove your identity this way.",
+        "subHeading": "If you’re not sure you’ll be able to prove your identity online",
+        "paragraph3": "You can prove your identity in person at a Post Office instead.",
+        "paragraph4": "This option takes longer. You’ll usually get an outcome within a day of going to the Post Office.",
+        "paragraph5": "You’ll need a UK or non-UK photo ID (such as a passport, driving licence, national identity card, or biometric residence permit).",
+        "paragraph6": "You’ll be asked to:",
+        "bullet3": "enter details from your photo ID on GOV.UK",
+        "bullet4": "go a Post Office where they’ll scan your ID and take a photo of you",
```
Comment (AmritSidhu) [inline src/locales/en/translation.json]: Looks like missing text on this line, "go a Post Office"...

File: src/locales/cy/translation.json [hunk 3]
```diff
@@ -275,6 +275,30 @@
...
       }
     },
+    "pageMultipleF2FDocCheck": {
+      "title": "Parhau profi eich hunaniaeth ar-lein",
+      "header": "Parhau profi eich hunaniaeth ar-lein",
+      "content": {
+        "paragraph1": "Gallwch barhau i brofi’ch hunaniaeth ar-lein os:",
+        "bullet1": "oes gennych basbort y DU neu drwydded yrru y DU",
+        "bullet2": "rydych yn gallu ateb rhai cwestiynau diogelwch am bethau fel eich contract ffôn symudol, ac unrhyw gyfrifon banc, cardiau credyd, benthyciadau neu forgeisi sydd gennych o bosibl",
+        "paragraph2": "Mae'n cymryd tua 10 munud i brofi eich hunaniaeth fel hyn.",
```
Comment (AmritSidhu) [inline src/locales/cy/translation.json]: Can you change ' to be ’. There are a few occurrences

File: src/views/ipv/page-f2f-multiple-doc-check.njk [hunk 4]
```diff
@@ -0,0 +1,69 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleName = 'pages.pageMultipleF2FDocCheck.title' | translate %}
```
Comment (AmritSidhu) [inline src/views/ipv/page-f2f-multiple-doc-check.njk]: Sorry to be a pain, can we change it to pageF2FMultipleDockCheck. Feel like pageMultipleF2FDocCheck sounds like there are multiple F2F related docs. And this way it matches what the file is called

Comment (shivanshuit914) [reply src/views/ipv/page-f2f-multiple-doc-check.njk]: No problem, Yes makes sense thanks

File: src/locales/en/translation.json (hunk 2 above)
Comment (shivanshuit914) [reply src/locales/en/translation.json]: hm, no I double checked it in ticket :) 

https://govukverify.atlassian.net/browse/PYIC-2876

Comment (AmritSidhu) [reply src/locales/en/translation.json]: Yeah this is a mistake in the ticket and screenshot attached. It should be "go to the Post Office..". I just checked the welsh translation and thats what it translates to. You can double check with content though

Comment (shivanshuit914) [reply src/locales/en/translation.json]: its list

Screenshot 2023-07-18 at 14 54 46

Comment (AmritSidhu) [reply src/locales/en/translation.json]: Yeah that second bullet point is wrong, it doesnt make sense. The welsh translations is correct but the english isnt.

**Review by blakeyp — APPROVED** [review]

**Pull request merged**